.env.example

```diff
@@ -12,8 +12,8 @@ DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
 DB_DATABASE=example_app
-DB_USERNAME=root
-DB_PASSWORD=
+DB_USERNAME=LHMUser
+DB_PASSWORD=LHMPassword
```

Comment:
You can rename the database, username and password to "learnhub"

Reply:
Additionally, you can create a .env.docker.example file and move everything that needs specific docker config there.

Reply:
yes, but we are talking about juniors onboarding, most be as simple as posible